clap-v3-utils/src/keypair.rs

```diff
@@ -370,22 +370,22 @@ impl DefaultSigner {
     }
 }
 
-pub(crate) struct SignerSource {
-    pub kind: SignerSourceKind,
+pub(crate) struct Source {
+    pub kind: SourceKind,
```

Comment:
are these renames strictly necessary?  the existing types are part of the public API, so even if so, we need to keep the old names around as deprecated symbols.

if not required, please revert.  otherwise, please factor the changes out to a separate PR for discussion

Reply:
Yeah, the renames are not necessary and it is just to have better names since we are now also deriving encryption keys, which are not signers (`SignerSource` --> `Source`). I wouldn't have made the changes if `SignerSource` had public visibility, but it seemed like it was limited to the crate only. But maybe it is better to have consistency between `clap-utils` and `clap-v3-utils`, so I will revert.